### PR TITLE
Deprecates static homepage.  Conditionally restricts finder

### LINKED ISSUE #2224: Campaign Finder feature flag should only toggle Finder UI
Right now, we're toggling between the old homepage design and the "new" homepage design with the Campaign Finder feature flag. This should _only_ toggle the Finder UI now that we're rendering homepage elements from the Home node. We can also get rid of all the old SCSS.

### PR DESCRIPTION
**Pull request by desmondmorris**
#### What's this PR do?
- Removes the old static homepage markup and conditional logic
- Removes associated stylesheets and references
- Conditionally restricts the campaign finder display via the show_campaign_finder theme setting
- Sets template variables for campaign finder & sponsor toggles to remove db calls from the tpl.
#### Where should the reviewer start?

The most significant changes appear here: https://github.com/desmondmorris/dosomething/blob/369157275c10b887fd43b5dc485d24df2e01508f/lib/themes/dosomething/paraneue_dosomething/templates/home/node--home.tpl.php
#### How should this be manually tested?
- Pull down my branch and clear cache.
- Visit theme admin page for paraneue_dosomething.  Toggle the campaign finder & sponsors flags and check the homepage.
#### Any background context you want to provide?

Since moving to a homepage node, there is no real reason to continue lugging around the static homepage markup.  It has been necessary to date to keep the international platform happy.

This work is necessary to grant privileged users on international affiliate sites the ability to remove the campaign finder from the homepage.
#### What are the relevant tickets?

Fixes #2224

cc: @DFurnes @weerd - please double check this and test in your env if you get a chance.

### CODE REVIEW

**Comment by weerd** [comment]
Checking code and looks good :+1: I can aim to pull down code and do a more thorough test later today.

**Comment by blisteringherb** [comment]
@desmondmorris  This PR is awesome.  I'm stealing this template.

**Comment by weerd** [comment]
Yeah, I agree w/ @blisteringherb, excellent organization @desmondmorris :dancer:

**Comment by desmondmorris** [comment]
@weerd were you able to give this a whirl?

File: lib/themes/dosomething/paraneue_dosomething/templates/home/node--home.tpl.php [hunk 1]
```diff
@@ -104,77 +106,5 @@
...
-      <i title="AARP" class="sprite sprite-aarp"></i>
-      <i title="Sprint" class="sprite sprite-sprint"></i>
-      <i title="H&amp;M" class="sprite sprite-hm"></i>
-      <i title="Salt" class="sprite sprite-salt"></i>
-      <i title="American Express" class="sprite sprite-amex"></i>
-      <i title="Google" class="sprite sprite-google"></i>
-    </p>
-  </div>
 </div>
 <?php endif; ?>
```
Comment (DFurnes) [inline lib/themes/dosomething/paraneue_dosomething/templates/home/node--home.tpl.php]: Looks like this guy missed the purge. :fire:

**Comment by DFurnes** [comment]
Tested on my local, see code comment on the `endif`. Once that's fixed, it all looks good. :+1: 

We'll also need to fix up how the image lazy-loading is initialized, since `main.js` is looking for the `.js-campaign-finder` guy to exist before running any of the code to for that. Can address that in a separate issue.

**Comment by weerd** [comment]
Looks good! Apart from what @DFurnes mentioned w/ the straggling `endif`, everything looks good! :+1:

**Comment by desmondmorris** [comment]
thanks @DFurnes @weerd!  That hanging endif has been terminated.

**Pull request merged**